commit 0f5139868a087d0af57f1c8bb8eb5b71e7cee470
Author: marc.schweitzer <marc.schweitzer@ircad.u-strasbg.fr>
Date:   Mon Jul 4 10:21:12 2016 +0200

    Fixed compilation of vtkEGLRenderingWindow when compiling for android with shared library

diff --git a/Rendering/OpenGL2/CMakeLists.txt b/Rendering/OpenGL2/CMakeLists.txt
index 35f57fb..5067ac9 100644
--- a/Rendering/OpenGL2/CMakeLists.txt
+++ b/Rendering/OpenGL2/CMakeLists.txt
@@ -332,7 +332,8 @@ endif()
 
 if(ANDROID)
   target_link_libraries(${vtk-module}
-    LINK_PUBLIC ${OPENGL_egl_LIBRARY})
+    LINK_PUBLIC ${OPENGL_egl_LIBRARY} android
+    )
 endif()
 
 if (APPLE_IOS)
diff --git a/Rendering/OpenGL2/vtkEGLRenderWindow.cxx b/Rendering/OpenGL2/vtkEGLRenderWindow.cxx
index 286a9fa..1be8372 100644
--- a/Rendering/OpenGL2/vtkEGLRenderWindow.cxx
+++ b/Rendering/OpenGL2/vtkEGLRenderWindow.cxx
@@ -29,6 +29,9 @@
 #include <sstream>
 #include <EGL/egl.h>
 
+#ifdef ANDROID
+#include <android/native_window.h>
+#endif
 
 
 namespace
